scripts/ecnconfig

```diff
@@ -0,0 +1,121 @@
+                line = [field, profile_data[field]]
+                config.append(line)
+            print(tabulate(config) + "\n")
+        if self.verbose:
+            print("Total profiles: %d" % len(wred_tables))
+
+    def set_wred_threshold(self, profile, threshold, value):
+        field = WRED_CONFIG_FIELDS[threshold]
+        if self.verbose:
+            print("Setting %s value to %s" % (field, value))
+        self.db.set_entry(WRED_PROFILE_TABLE_NAME, profile, {field: value})
+
+def main():
+    parser = argparse.ArgumentParser(description='Show and change ECN WRED configuration',
+                                     version='1.0.0',
+                                     formatter_class=argparse.RawTextHelpFormatter)
+    # group = parser.add_mutually_exclusive_group()
+    parser.add_argument('-l', '--list', action='store_true', help='show ECN WRED configuration')
+    parser.add_argument('-p', '--profile', type=str, help='specify WRED profile name', default=None)
+    parser.add_argument('-gmin', '--green-min', type=str, help='set min threshold for packets marked \'green\'', default=None)
+    parser.add_argument('-gmax', '--green-max', type=str, help='set max threshold for packets marked \'green\'', default=None)
+    parser.add_argument('-ymin', '--yellow-min', type=str, help='set min threshold for packets marked \'yellow\'', default=None)
+    parser.add_argument('-ymax', '--yellow-max', type=str, help='set max threshold for packets marked \'yellow\'', default=None)
+    parser.add_argument('-rmin', '--red-min', type=str, help='set min threshold for packets marked \'red\'', default=None)
+    parser.add_argument('-rmax', '--red-max', type=str, help='set max threshold for packets marked \'red\'', default=None)
+    parser.add_argument('-vv', '--verbose', action='store_true', help='Verbose output', default=False)
+    args = parser.parse_args()
+
+    try:
+        ecn = EcnConfig(args.verbose)
+        if args.list:
+            if len(sys.argv) > (3 if args.verbose else 2):
+                raise Exception("Input arguments error. No set options allowed when -l[ist] specified")
+            ecn.list()
+        elif args.profile:
+            if len(sys.argv) < (5 if args.verbose else 4):
+                raise Exception("Input arguments error. Specify at least one threshold parameter to set")
+            # the following parameters can be combined in one run
+            if args.green_max:
+                ecn.set_wred_threshold(args.profile, "gmax", args.green_max)
+            if args.green_min:
+                ecn.set_wred_threshold(args.profile, "gmin", args.green_min)
+            if args.yellow_max:
+                ecn.set_wred_threshold(args.profile, "ymax", args.yellow_max)
+            if args.yellow_min:
+                ecn.set_wred_threshold(args.profile, "ymin", args.yellow_min)
+            if args.red_max:
+                ecn.set_wred_threshold(args.profile, "rmax", args.red_max)
+            if args.red_min:
```

Comment:
Maybe deleting comments is not the best thing.

I figured out that args are returning values in strings, so we don't have a risk of ignoring integer value 0 here.